js/actions/_admin/foxx/app.js

```diff
@@ -80,9 +80,11 @@ function resolveAppInfo (appInfo, refresh) {
   if (/^git:/i.test(appInfo)) {
     const splitted = appInfo.split(':');
     const baseUrl = process.env.FOXX_BASE_URL || 'https://github.com/';
+    FoxxManager.validateInstallUrl(baseUrl);
     return {source: `${baseUrl}${splitted[1]}/archive/${splitted[2] || 'master'}.zip`};
   }
   if (/^https?:/i.test(appInfo)) {
+    FoxxManager.validateInstallUrl(appInfo);
```

Comment:
Do we need this check here? It's a small optimization to avoid proxying an invalid request but this route is deprecated anyway.